### ISSUE TITLE
Unixcoder fine tune on POJ-104, how to use it for inference?

### ISSUE THREAD

**Issue by HamzaBale** [microsoft/CodeBERT#322]
Hi everybody, yesterday i executed the script that you can find [here](https://github.com/microsoft/CodeBERT/tree/master/UniXcoder/downstream-tasks/clone-detection/POJ-104). This process returned a file called "model.bin" in the directory "saved_models", now i want to use this model in order to create embeddings for source code, in evaluation mode like i used to do here:"
model = UniXcoder("microsoft/unixcoder-base")
model.to(device)
model.eval()"
How to do that?
